helm/chart-operator/templates/00-service-account.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
```

Comment:
@tomahawk28 Why did we start using this `00-service-account.yaml` naming?  We don't use this now so why do we need it?

Reply:
@rossf7 I was following the naming from new helm chart in aws-operator, and it's easier to figure out the order of installing components. WDYT?

Reply:
@rossf7 I deleted all the numbers :) PTAL!